lib/run_loop.rb

```diff
@@ -301,4 +306,16 @@ def self.detect_automator(options, xcode, device)
       RunLoop.default_automator(xcode, device)
     end
   end
+
+  def self.keyboard_hidden?
+    File.file?(preferences_plist_path) && RunLoop::PlistBuddy.new.plist_read('HardwareKeyboardLastSeen', preferences_plist) == 'true'
```

Comment:
Minor: Would consider putting this on two lines to increase readability

Reply:
Actually, I think you can drop the `== true` and it will still work. But I guess it's more explicit like this. Your choice